modules/ROOT/pages/changelogs.adoc

```diff
@@ -1,6 +1,13 @@
 :description: This page lists all changes to status codes per Neo4j version.
 = Changes to status codes per Neo4j version
 
+== Neo4j 5.23
+
+From version 5.23, Neo4j supports GQLSTATUS codes in addition to the existing Neo4j `ClientNotification` codes when used with the latest drivers and configured to use both the Neo4j and GQLSTATUS framework. +
```

Comment:
Yeah, in this manual we don't have anything about how the backend works. I added something in the compatibility section because, otherwise, the whole compatibility concept would be out of context. I'll think about it and see if we can have something here, or do we want to have a section about how the whole system works or a technical diagram?